YIF.Core.Domain/Repositories/SpecialtyRepository.cs

```diff
@@ -28,9 +28,11 @@ public async Task<bool> Update(Specialty specialty)
              return await _context.SaveChangesAsync() > 0;
         }
 
-        public Task<bool> Delete(string id)
+        public async Task<bool> Delete(string id)
         {
-            throw new NotImplementedException();
+            var service = _context.Specialties.FirstOrDefault(x => x.Id == id);
```

Comment:
meaningful name, rename variable service  to specialty

Reply:
Done)